Snake/Snake.ino

```diff
@@ -62,35 +62,54 @@ void THE_END()
 {
   Serial.println("Vesztettél!!!");
 
+  for (int i = 0; i < kigyoHossza + 1; i++)
+  {
+    lc.setLed(0, kigyo[i].x, kigyo[i].y, false);
+    delay(100);
+  }
+
+  for (int i = 0; i < kigyoHossza + 1; i++)
+  {
+    lc.setLed(0, kigyo[i].x, kigyo[i].y, true);
+    delay(100);
+  }
+
+  for (int i = 0; i < kigyoHossza + 1; i++)
+  {
+    lc.setLed(0, kigyo[i].x, kigyo[i].y, false);
+    delay(100);
+  }
+
+  for (int i = 0; i < kigyoHossza + 1; i++)
+  {
+    lc.setLed(0, kigyo[i].x, kigyo[i].y, true);
+    delay(100);
+  }
+
   while (1)
   {
```

Comment:
Ha jól gondolom, itt a teljes kijelzőn átfutó "hullámzást" módosítottad egy kicsit. Nagyon jó effekt, ügyesen megcsináltad, és jól is fog működni, de mégis van vele kapcsolatban két kérdésem:
1. Mi a pontos célod, hogy szeretnéd, hogy kinézzen ez? A kódod alapján mindig csak egyetlen oszlop lesz felkapcsolva, ez fog egy irányba futkározni a kijelzőn, valahogy így: 
![pulllreq1](https://github.com/PBotond/Robotika_stuff/assets/24439376/db8c45f4-c1da-4da6-9cf4-36a536e71c88)
Ez egy tök jó effektus, ha ez volt a cél, tekintsd ezt az első kérdést tárgytalannak 😅.

Ha azt akartad, hogy az egész kijelző oszloponként sorban felkapcsolódjon, aztán oszloponként sorban le, ahogy a lenti képen, akkor kicsit máshogy kell szervezni a ciklusokat.
![pulllreq2](https://github.com/PBotond/Robotika_stuff/assets/24439376/b376f72c-b135-4e0e-80f7-d49f4a72cb91)

2. Ha az első opciót szeretnéd, akkor nincs szükség mind a négy for ciklusra, háromból is meg lehet oldani. Jelenleg egy oszlop felkapcsolása után mindig sorba mész az összes többin és azokat is lekapcsolod, bár biztosan nincsenek felkapcsolva (ezért gondolkoztam el az előző kérdésben, hogy mit szerettél volna pontosan 😃). Ügyesen kihasználod, hogy a belső ciklusból látható, elérhető, használható a külső ciklus ciklusváltozója. Szerintem adj eltérő neveket az összes ciklusváltozónak (az i,j,k,l,stb. betűk mellett használhatsz beszédesebb neveket, pl. oszlop, sor, stb. is 😉), úgy könnyebb lesz átlátni. Ha nem érted, hogy mire gondolok, próbáld ki, hogy a setLed függvényhívások helyén elküldöd a soros portra a ciklusváltozók értékét és hogy fel/lekapcsolod az adott LED-et. Ehhez a végtelen ciklust érdemes ideiglenesen egy egyszeri lefutásra cserélni.